PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1493,38 +1493,42 @@ intptr_t CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 			if (LOWORD(wParam) == WA_ACTIVE || LOWORD(wParam) == WA_CLICKACTIVE)
 			{
 				Sci_CharacterRangeFull cr = (*_ppEditView)->getSelection();
-				intptr_t nbSelected = cr.cpMax - cr.cpMin;
+				LRESULT nbSelected = (*_ppEditView)->execute(SCI_COUNTCHARACTERS, cr.cpMin, cr.cpMax);

```

Comment:
I have some weird behaviour (the count of character not exact) while testing this PR. I'm not sure it's due to SCI_COUNTCHARACTERS. 
However, since the goal of the PR is adding "disable In selection auto-checked" feature, not change current behaviour, please restore `intptr_t nbSelected = cr.cpMax - cr.cpMin;`

Reply:
I agree to defer related discussion to a later point on this; restoring position subtraction code for now.